app/code/community/Kirchbergerknorr/GoogleTagManager/Model/Observer.php

```diff
@@ -42,13 +43,18 @@ public function updateLayout(Varien_Event_Observer $observer)
 
         if ($configHelper->getGoogleTagManagerIsActive()) {
 
+            $this->layoutUpdateSnippet =
```

Comment:
It should be sufficient that this is only set to a local variable as the property is never reused.